src/components/Launch/LaunchWorkflowForm/DatetimeInput.tsx

```diff
@@ -8,34 +9,33 @@ import { Moment, utc as moment } from 'moment';
 import * as React from 'react';
 import { InputProps } from './types';
 
-function defaultDate() {
-    return moment()
-        .startOf('day')
-        .toISOString();
-}
-
 /** A form field for selecting a date/time from a picker or entering it via
  * keyboard.
  */
 export const DatetimeInput: React.FC<InputProps> = props => {
     const { label, helperText, onChange, value: propValue } = props;
-    const value = typeof propValue === 'string' ? propValue : defaultDate();
+    const value =
+        typeof propValue === 'string' && propValue.length > 0
+            ? propValue
+            : null;
 
     const handleChange = (
         dateValue: Moment | null,
         stringValue?: string | null
     ) => {
-        if (stringValue != null) {
-            onChange(stringValue);
-        } else if (dateValue !== null) {
+        if (dateValue && dateValue.isValid()) {
             onChange(dateValue.toISOString());
+        } else if (stringValue != null) {
```

Comment:
`!==` or are you deliberately coercing? If the latter, would you just want `else if (stringValue)`?

Reply:
Deliberately coercing. I want to allow empty string through, but not null/undefined.

Reply:
Doh, although I do check above for empty string and change it to a date anyway...